operations/messaging-and-math/how-to-guides-for-m-and-m/how-to-create-release-announcements.md

```diff
@@ -107,6 +109,35 @@ The Release Manager \(Amy Blais\) owns the release announcement, including the f
 4. Do not use internal names of features, such as "Unread toasts", "Deep linking"
 5. Say "You" instead of "Users" to talk directly to the reader
 
+## Scoring Guidelines
+
+<table>
```

Comment:
Not sure how to fix the table

Reply:
@amyblais I fixed it with https://github.com/mattermost/mattermost-handbook/pull/210/commits/b510eda5871672e0bcc233c4e2b16ee7b3db968d

(It needed a couple more `<td></td>` tags so that it displays empty cells for marketing and sysadmin representative columns)

Reply:
I also used `<p></p>` tags to create new lines for the three scores